Update README.md (#6284)
swagger-api/swagger-ui: docs/customization/plugin-api.md

#### Reducers

> Reducers take a state (which is an [Immutable.js map](https://facebook.github.io/immutable-js/docs/#/Map)) and an action and return a new state.

Reviewer: ```suggestion
Reducers take a state (which is an [Immutable.js map](https://facebook.github.io/immutable-js/docs/#/Map)) and an action, then returns a new state.
```

Selectors reach into their namespace's state to retrieve or derive data from the state.

> They're an easy way to keep logic in one place and are preferred over passing state data directly into components.

Reviewer: ```suggestion
They're an easy way to keep logic in one place, and are preferred over passing state data directly into components.
```

}
```

> You can also use the Reselect library to memoize your selectors, which is recommended for any selectors that will see heavy use since Reselect automatically memoizes selector calls for you:

Reviewer: ```suggestion
You can also use the Reselect library to memoize your selectors, which is recommended for any selectors that will see heavy use, since Reselect automatically memoizes selector calls for you:
```

#### Wrap-Actions

> Wrap Actions allow you to override the behavior of the action in the system.

Reviewer: `an` is more correct, as there are many possible actions.

Author: ok

> Wrap Actions allow you to override the behavior of the action in the system.

Reviewer: ```suggestion
Wrap Actions allow you to override the behavior of an action in the system.
```
